pkg/api/types.go

```diff
@@ -310,6 +312,8 @@ type PersistentVolumeList struct {
 	Items                []PersistentVolume `json:"items"`
 }
 
+// +genclient=true
+
 // PersistentVolumeClaim is a user's request for and claim to a persistent volume
 type PersistentVolumeClaim struct {
```

Comment:
For client-gen,  the tag must be in the comment before the type's doc. This prevents the tag from showing up in godoc or our API docs (definitions.html).

Users of go2idl can choose whether the tag should exist in the immediate comment block or the comment block before it.